src/api/rollups.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: home.njk
+title: FAC API rollup tutorial
+meta:
+  name: FAC API rollup tutorial
+  description: A tutorial focused on the rollup fields in the API
+rollups:
+    ALN:
+        rolled: CFDA
+        orig:  CFDA table
+        criteria: A combination of agency and program number
+        output: A CFDA/ALN number
+        censusdesc: FIXME
+        code: https://github.com/GSA-TTS/fac-api-examples/tree/main/rollups/01-aln
+    COGOVER:
+        rolled: FIXME
+        orig: FIXME
+        criteria: FIXME
+        output: FIXME
+        censusdesc: FIXME
```

Comment:
I agree, but this is probably something we can/should come back to later since it's so deeply embedded right now and we're hoping to get this merged very soon.